File: 2015/python/dia_1/dia_1.py
with open('input.txt', 'r') as file:
	inputFile = file.read()

	print(inputFile) 


	floor = 0
	position = 1

	for i in inputFile:
		if i == '(':
			floor += 1
			
		elif i == ')':
			floor -= 1
			

		if floor == -1 :
			print("enter the basement in position: ",position)
			break
		print("position: ",position)
		position += 1;

File: 2015/java/dia_1/Dia1.java
import java.io.File;
import java.io.IOException;
import java.util.Scanner;

public class Dia1 {
	public static void main (String[] args){
		int floor = 0;

		try{
			File file = new File("/Users/rosasainz/Documents/prog/rosa/versionado/adventOfCode_mySolutions/2015/java/dia_1/input.txt");
			Scanner scanner = new Scanner(file);

			while(scanner.hasNextLine()){
				String line = scanner.nextLine();
				System.out.print(line);

				for(int i=0; i < line.length(); i++){
					char c = line.charAt(i);
					if( c == '(' ){
						floor += 1;
					} else if(c == ')') {
						floor -= 1;
					}
				}
			}
			System.out.print("floor: "+floor);

			scanner.close();
		} catch(IOException e){
			System.out.print("No se puede leer el archivo");
		}

	}
}
